python/pyspark/pandas/tests/data_type_ops/test_boolean_ops.py

```diff
@@ -132,7 +128,7 @@ def test_floordiv(self):
         self.assertRaises(TypeError, lambda: b_psser // b_psser)
         self.assertRaises(TypeError, lambda: b_psser // True)
 
-        self.assert_eq(b_pser // pdf["float"], b_psser // psdf["float"])
+        self.assert_eq((b_pser // pdf["float"]).astype("int"), b_psser // psdf["float"])
```

Comment:
According to https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.floordiv.html, as well as, integers should always be returned.